web/geo/generate-exchanges-to-exclude.js

```diff
@@ -20,11 +20,23 @@ const generateExchangesToIgnore = (OUT_PATH, fc) => {
     const split = key.split('->');
     const zoneOne = split[0].slice(0, 2);
     const zoneTwo = split[1].slice(0, 2);
-    if (zoneOne === zoneTwo && countryKeysToExclude.includes(key.slice(0, 2))) {
+    if (key.length > 6 && (countryKeysToExclude.includes(zoneOne) || countryKeysToExclude.includes(zoneTwo))) {
       return key;
     }
   });
-  const exchanges = { exchangesToExclude: unCombinedExchanges };
+
+  const countryExchangesWithSubzones = Object.keys(exchangeConfig).filter((key) => {
+    const split = key.split('->');
+    const zoneOne = split[0].slice(0, 2);
```

Comment:
I assume we are slicing here to get the country code?
If so just be aware that for Australia we use AUS as the country code, now the split works here but it won't match zone config keys.

Reply:
Nice catch, I'll need to rethink this solution

Reply:
You should be able to split it at `-` and just grab the first part of the array.

Reply:
I think I might re do the logic based on the zones config to make it a bit more of a direct mapping, your catch has highlighted to me why it's a bit flawed to do it this way.

Reply:
Note: you might want to look at zones.json which has the `subZones` key which enables you to figure out which zones have subzones.